Give attribute dialogs more space

HDinger opened the pull request:
Request to implement a quick win to give the hierachical attributes more space. Otherwise the tree view is rather squeezed

HDinger committed:
Give attribute dialogs more space

ulferts reviewed (approved):
Good to merge once specs agree.

cbliard committed:
Try to fix a flaky spec

spec is spec/features/projects/project_custom_fields/overview_page/dialog/inputs_spec.rb:702
run is https://github.com/opf/openproject/actions/runs/19696080834/job/56421276049?pr=21163

Not entirely sure what's going on, but on the screenshot the field looks
active, so it's clicked, but for some reason the options are not
visible.

Try to work around the issue by waiting for the options to be visible,
and click the input again if not. Check every 0.25 seconds and try for 3
seconds before giving up.

cbliard committed:
Fix flaky spec

spec is spec/features/projects/project_custom_fields/overview_page/dialog/inputs_spec.rb:702
run is https://github.com/opf/openproject/actions/runs/19700308113/job/56434713171?pr=21163

When opening the options of the dropdown, the field is clicked. If unlucky, it can click on the "X" of an option already selected and remove it. When this happens, it does not open the dropdown (this explains why the previous run was flaky).

The default implementation in both Cuprite and Selenium is to click in
the middle of the element. I don't understand why we can't reproduce it
though.

Fix is to click the arrow directly.

HDinger commented:
It is green 🥳 Thanks a lot @cbliard 🙇
